src/main/scala/viper/gobra/reporting/VerifierError.scala

```diff
@@ -246,6 +246,11 @@ case class MapMakePreconditionError(info: Source.Verifier.Info) extends Verifica
   override def localMessage: String = s"The provided length to ${info.origin.tag.trim} might be negative"
 }
 
+case class ShiftPreconditionError(info: Source.Verifier.Info) extends VerificationError {
+  override def localId: String = "shift_precondition_error"
+  override def localMessage: String = s"The shift count in ${info.origin.tag.trim} might be negative"
```

Comment:
Is `count` a common terminology? I would otherwise prefer `amount`
```suggestion
  override def localMessage: String = s"The shift amount in ${info.origin.tag.trim} might be negative"
```

Reply:
Count is the default terminology of the Go compiler. For example,
```go
package main

const N = 1 << 513
```
leads to the compiler error
```
./prog.go:3:13: shift count too large: 513
```

Reply:
Okay, then ignore my comment